example-local/autorun/autorun.sh

```diff
@@ -135,6 +145,26 @@ verify_installed()
   return 0
 }
 
+check_docker_compose_version() {
+  local version=""
+
+  if [ "${DOCKER_COMPOSE}" == "docker-compose" ]; then
+    version=$(${DOCKER_COMPOSE} --version | cut -d" " -f3 | sed "s/,//")
+  else
+    version=$(${DOCKER_COMPOSE} version | cut -d" " -f4)
+  fi
+
+  local compare=$(semver compare "${version}" "${COMPOSE_MIN_VERSION}")
+  if [ "${compare}" == "1" ] || [ "${compare}" == "0" ]; then
```

Comment:
can you add a comment to link the place where you find the source of the version discriminant ? 
And explain why we do this (in 2 line) ?